### ISSUE TITLE
Save encoder and decoder of an autoencoder respectively

### ISSUE THREAD

**Issue by loganXcode** [pytorch/pytorch#3709]
I have trained an autoencoder, I want to save its encoder and decoder respectively. 

In other words, I want to save some layers in the model instead of all layers. But it seems that torch.save can only save the entire model.

Can anyone give me some suggestions or a simple example?

Thank you so much.

class autoencoder(nn.Module):
    def __init__(self):
        super(autoencoder, self).__init__()
        self.encoder = nn.Sequential(
            nn.Conv2d(1, 16, 3, stride=3, padding=1),  # b, 16, 10, 10
            nn.ReLU(True),
            nn.MaxPool2d(2, stride=2),  # b, 16, 5, 5
            nn.Conv2d(16, 8, 3, stride=2, padding=1),  # b, 8, 3, 3
            nn.ReLU(True),
            nn.MaxPool2d(2, stride=1)  # b, 8, 2, 2
        )
        self.decoder = nn.Sequential(
            nn.ConvTranspose2d(8, 16, 3, stride=2),  # b, 16, 5, 5
            nn.ReLU(True),
            nn.ConvTranspose2d(16, 8, 5, stride=3, padding=1),  # b, 8, 15, 15
            nn.ReLU(True),
            nn.ConvTranspose2d(8, 1, 2, stride=2, padding=1),  # b, 1, 28, 28
            nn.Sigmoid()
        )

    def forward(self, x):
        x = self.encoder(x)
        x = self.decoder(x)
        return x

**Comment by LukasMosser** [comment]
Can you post this in the discussion forums? This is not an issue with the code itself.

**Comment by loganXcode** [comment]
OKay, thank U
